experiments/AMIP/modular/components/land/bucket_init.jl

```diff
@@ -188,27 +189,35 @@ function bucket_init(
     α_snow = FT(0.8) # snow albedo
     if albedo_type == "map_static" # Read in albedo from static data file (default type)
         # By default, this uses a file containing bareground albedo without a time component. Snow albedo is specified separately.
+        if ClimaComms.iamroot(comms_ctx)
+            albedo = BulkAlbedoStatic{FT}(regrid_dirpath, α_snow = α_snow, comms = comms_ctx)
+        end
+        ClimaComms.barrier(comms_ctx)
         albedo = BulkAlbedoStatic{FT}(regrid_dirpath, α_snow = α_snow, comms = comms_ctx)
     elseif albedo_type == "map_temporal" # Read in albedo from data file containing data over time
         # By default, this uses a file containing linearly-interpolated monthly data of total albedo, generated by CESM2's land model (CLM).
+        if ClimaComms.iamroot(comms_ctx)
+            albedo = BulkAlbedoTemporal{FT}(regrid_dirpath, date_ref, t_start, space)
+        end
+        ClimaComms.barrier(comms_ctx)
```

Comment:
See [interactive_debug.yml](https://github.com/CliMA/ClimaCoupler.jl/pull/491/files#diff-122ff1b60cd05015f670229a0b6ce547f76cf250e02f62d735eae69b5fe5cacd)

Reply:
Is this one ok to close, or was there a question? :)

Reply:
Oops - wrong line link :S Same comment as above w.r.t. sync -> Since you started #470 this is resolved.